spec/support/rails_helper.rb

```diff
@@ -9,6 +9,10 @@ def secrets_class
     ActiveSupport::OrderedOptions
   end
 
+  def credentials_class
```

Comment:
we positive that on all our Rails versions the secrets/credentials thingies exposed from the `config` are `OrderedOptions`?

Reply:
I am seeing various resources online indicating that rails uses OrderedOptions for secrets/credentials dating back to rails 4 even, so I am betting on the fact that this will not cause issues. If so, I'm happy to receive an issue/pr about it. :)